docs/components.md

```diff
@@ -32,10 +32,11 @@ support at this time). While in alpha, breaking changes may be introduced in a
 new release. In addition, any of these components may be removed prior to the
 1.0 release.
 
-| Receivers      | Processors | Exporters | Extensions |
-| :-------:      | :--------: | :-------: | :--------: |
-| carbon         |            | kafka     |            |
-| collectd       |            |           |            |
-| filelog        |            |           |            |
-| kafka          |            |           |            |
-| statsd         |            |           |            |
+| Receivers            | Processors | Exporters | Extensions |
+| :-------:            | :--------: | :-------: | :--------: |
+| carbon               |            | kafka     |            |
+| collectd             |            |           |            |
+| filelog              |            |           |            |
+| kafka                |            |           |            |
+| kafkametricsreceiver |            |           |            |
```

Comment:
Looks like that worked but you gotta update this one too :P Then I think we're good.

Reply:
oops, sorry! Thank you again! Fixed.